assets/js/googlesitekit/api/index.js

```diff
@@ -21,21 +21,21 @@ let cachingEnabled = true;
 /**
  * Make a request to a WP REST API Site Kit endpoint.
  *
- * @param {string} type        The data to access. One of 'core' or 'modules'.
- * @param {string} identifier  The data identifier, eg. a module slug like `'search-console'`.
- * @param {string} datapoint   The endpoint to request data from.
- * @param {Object} queryParams Query params to send with the request.
- * @param {Object} data        Request data to send.
- * @param {Object} useCache    Set to `true` to use the caching. Caching is only used for `GET` requests.
+ * @param {string}  type                The data to access. One of 'core' or 'modules'.
+ * @param {string}  identifier          The data identifier, eg. a module slug like `'search-console'`.
+ * @param {string}  datapoint           The endpoint to request data from.
+ * @param {Object}  options             Options to pass to the request
+ * @param {number}  options.cacheTTL    The oldest cache data to use, in seconds.
+ * @param {Object}  options.data        Request data to send.
```

Comment:
Maybe this should say something about `Request body data`, to clarify? I think it would furthermore help to name this `formData` or `formParams` or `bodyParams` or something here, to differentiate between our generic `data` name that we use regardless of whether it's a GET or POST request.

In other words:
* `data` should be used for the generic parameters a GET or POST request accepts. All our API's datapoints expect `data` already.
* `queryParams` should be a more low-level thing used e.g. in this `siteKitRequest` function - essentially for `data` that is sent via query parameters.
* The above `bodyParams` (or other name) should be similarly a more low-level thing used here, for `data` that is sent via body parameters.

Both `set` and `get` should then accept a parameter called `data`, and pass it to `siteKitRequest` as either `queryParams` (for GET) or `bodyParams` (for POST). This would make it more consistent with our namings so far and differentiate high-level function parameter names from the way it is sent (low-level API calls).